plugins/controlboard/src/ControlBoardDriver.cpp

```diff
@@ -1144,13 +1159,120 @@ bool GazeboYarpControlBoardDriver::setPIDsForGroup_IMPEDANCE(std::vector<std::st
     }
 }
 
-bool GazeboYarpControlBoardDriver::setMinMaxImpedance()
+bool GazeboYarpControlBoardDriver::FindMotorControlGroup(yarp::os::Bottle& motorControlGroup_bot) const
 {
+    if (!m_pluginParameters.check("WRAPPER"))
+    {
+        yError()<<"Missing WRAPPER group";
+        return false;
+    }
+
+    if (!m_pluginParameters.findGroup("WRAPPER").check("networks"))
+    {
+        yError()<<"Missing networks group";
+        return false;
+    }
 
     yarp::os::Bottle& name_bot = m_pluginParameters.findGroup("WRAPPER").findGroup("networks");
     std::string name = name_bot.get(1).toString();
 
-    yarp::os::Bottle& kin_chain_bot = m_pluginParameters.findGroup(name);
+    motorControlGroup_bot = m_pluginParameters.findGroup(name);
+    return true;
+}
+
+bool GazeboYarpControlBoardDriver::setPositionsToleranceLinear()
+{
+    yarp::os::Bottle kin_chain_bot;
+    if (!FindMotorControlGroup(kin_chain_bot))
+        return false;
+
+    if (!kin_chain_bot.check("positionToleranceLinear")) {
+        yWarning()<<"No positionToleranceLinear value found in ini file, default one will be used!";
+        yDebug()<<"positionToleranceLinear: [ "<<m_robotPositionToleranceLinear<<" ]";
+        return true;
+    }
+
+    yInfo()<<"positionToleranceLinear param found!";
+    yarp::os::Bottle& positionToleranceLinear_bot = kin_chain_bot.findGroup("positionToleranceLinear");
+    if (static_cast<size_t>(positionToleranceLinear_bot.size()) != 2) {
+        yError()<<"Invalid number of params:positionToleranceLinear:"<<static_cast<size_t>(positionToleranceLinear_bot.size());
+        return false;
+    }
+
+    auto tmp=positionToleranceLinear_bot.get(1);
+    if (!tmp.isDouble())
+    {
+        yError()<<"Invalid param type:positionToleranceLinear";
+        return false;
+    }
+
+    m_robotPositionToleranceLinear=tmp.asDouble();       
+
+    yDebug()<<"positionToleranceLinear: [ "<<m_robotPositionToleranceLinear<<" ]";
```

Comment:
I think we can remove this print.

Reply:
I prefer to keep, also because is present also in the other methods

Reply:
I see, I would want to remove also the others because the output of Gazebo is extremly confusing but you are right, consistency wins. : ) 
We should eventually move to have some verbosity option to enable/disable this prints at runtime, either using the `verbose` option of the YARP device or directly the Gazebo verbosity level. 
See https://github.com/robotology/gazebo-yarp-plugins/issues/87 and https://github.com/robotology/gazebo-yarp-plugins/pull/263 .